scripts/startUserMigration.js

```diff
@@ -0,0 +1,44 @@
+const { program } = require('commander');
+const appPromise = require('../src/app');
+const { error } = require('../src/logger');
+const { schoolModel } = require('../src/services/school/model');
+
+program
+	.option('-s, --systemId <value>', 'id of the system, which will be added to the schools (required argument)')
+	.option(
+		'-S, --schoolIds <value...>',
+		'ids of the schools that should be migrated. If not given all schools with a official school number will be migrated.',
+	);
+
+program.parse();
+
+const options = program.opts();
+
+const startUserMigration = async () => {
+	const { systemId } = options;
+	const schoolsToMigrate = options.schoolIds;
+	let schools;
+	if (!schoolsToMigrate) {
+		schools = schoolModel.find({ officialSchoolNumber: { $exists: true } });
+	} else {
+		schools = schoolModel.find({ _id: { $in: schoolsToMigrate } });
+	}
+	for await (const school of schools) {
+		try {
+			school.ldapSchoolIdentifier = school.officialSchoolNumber;
+			if (!school.systems.includes(systemId)) school.systems.push(systemId);
+			school.inUserMigration = true;
+			school.inMaintenanceSince = new Date();
+			// eslint-disable-next-line no-await-in-loop
+			await school.save();
```

Comment:
log school number, school id and that its migration mode has set

Reply:
We don't want to do this here. So if an error occured it will be easier to spot it. The script should also not run that long, that it is helpful to see progress like additional logging.

Reply:
we want to log only errors, otherwise would be difficult to spot errors in between non-error messages
also, the list of schools can be taken from db directly if anyone is interested in that